src/cbmc/cbmc_parse_options.cpp

```diff
@@ -685,9 +702,28 @@ int cbmc_parse_optionst::get_goto_program(
     {
       status() << "Reading GOTO program from file " << eom;
 
-      if(read_object_and_link(*it, symbol_table, goto_functions,
-                              get_message_handler()))
+      if(read_object_and_link(
+        *it,
+        symbol_table,
+        goto_functions,
```

Comment:
@kroening, is there a plan to generally move to goto_modelt instead of symbol_table+goto_functions?